x-pack/plugins/cloud_security_posture/public/components/page_template.test.tsx

```diff
@@ -4,15 +4,35 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-import React, { ComponentProps } from 'react';
+import React, { type ComponentProps } from 'react';
 import { render, screen } from '@testing-library/react';
 import Chance from 'chance';
+import { coreMock } from '../../../../../src/core/public/mocks';
+import { createStubDataView } from '../../../../../src/plugins/data_views/public/data_views/data_view.stub';
+import { CSP_KUBEBEAT_INDEX_PATTERN } from '../../common/constants';
+import { useKubebeatDataView } from '../common/api/use_kubebeat_data_view';
```

Comment:
IMO we should create a task to change the namings from `kubebeat` to `cloudbeat` there are a lot of mentions to this name which is, as far as I know, not the name we are actually using anymore.